### DISCUSSION TITLE
Movement Irregularities with WASD

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#485] [Q&A] [answered]
A user reported this today.

When moving Left, Right can override, but Left cannot override Right.

When moving Forward, Back can override, but Forward cannot override Back.

It appears related to the usage of the Negate modifier on the input values. Any way to resolve this that you can think of?

**Comment by Sixze** [comment, marked as answer]
This is a matter of the Enhanced Input plugin. There is a `Accumulation Behavior` setting in `IA_Als_Move` that is related to this.

**Reply by xkamronx** [reply]
Appreciate you, just going to change it and have them cancel eachother out then.
